packages/aws-rfdk/lib/core/lib/health-monitor.ts

```diff
@@ -211,6 +212,13 @@ export interface HealthMonitorProps {
    * @default: The VPC default strategy
    */
   readonly vpcSubnets?: SubnetSelection;
+
+  /**
+   * Security group to associate with the load balancer if it will be created.
```

Comment:
I looked and we do document that the `HealthMonitor` uses a load balancer, but for anyone who skips over that documentation might be confused when seeing this documentation. How about:

```suggestion
   * Security group for the health monitor. This is security group is associated with the health monitor's load
   * balancer.
```

Reply:
It is good but I just wanted to highlight here that this security group is used only when we create load balancer. 
We have option to pass LB and this SG will not be used then.

Reply:
If that is the case, perhaps we want to warn or error when a user supplies both properties to the `HealthMonitor`. As for the TSDoc, how about:

```suggestion
   * Security group for the health monitor. This is security group is associated with the health monitor's load
   * balancer. This security group is only used when a load balancer is not supplied to the health monitor.
```

Reply:
After discussion offline, we confirmed that `HealthMonitor` does not accept a load balancer as input.

Reply:
Resolved